src/energyplus/ForwardTranslator/ForwardTranslateControllerOutdoorAir.cpp

```diff
@@ -151,6 +159,17 @@ namespace energyplus {
       }
     }
 
+    // HumidistatControlZoneName
+    if (auto zone_ = modelObject.humidistatControlZone()) {
+      if (boost::optional<ZoneControlHumidistat> humidistat = zone_->zoneControlHumidistat()) {
+        idfObject.setString(openstudio::Controller_OutdoorAirFields::HumidistatControlZoneName, zone_->nameString());
+      } else {
+        LOG(Warn, modelObject.briefDescription() << " has a humidistat control zone " << zone_->nameString()
+                                                 << " without a zone control humidistat; humidistat control zone field will not be translated");
+        idfObject.setString(openstudio::Controller_OutdoorAirFields::HighHumidityControl, "No");
```

Comment:
HumidistatControlZoneName is required when HighHumidityControl is "Yes"; so if we can't set HumidistatControlZoneName, then HighHumidityControl must be set to "No".